### DISCUSSION TITLE
Remove the timer shown in fields when a file or path browser is opened

### DISCUSSION THREAD

**Discussion by quasiblob** [bmaltais/kohya_ss#295] [Ideas]
Why does the GUI show a timer (running number) when I open a file browser to browse for a path?

It doesn't actually matter if I see that, but it simply looks strange to see such a feature in web tech based UI.

**Comment by bmaltais** [comment]
If you know how to do that I will gladly add it. It is annoying me too... But I am not sure how to prevent the timer from happening.

**Reply by bmaltais** [reply]
OK. Figured it out. Removed in latest release.
